CDP4ShellDialogs/ViewModels/ModelOpeningDialogViewModel.cs

```diff
@@ -144,6 +247,45 @@ private async Task ExecuteSelect()
             this.DialogResult = new BaseDialogResult(true);
         }
 
+        /// <summary>
+        /// Executes the SelectActive command
+        /// </summary>
+        /// <returns>
+        /// The <see cref="Task"/>.
+        /// </returns>
+        private async Task ExecuteSelectActive()
+        {
+            var stopWatch = Stopwatch.StartNew();
+            var activeIterationSetupRow = this.SelectedRowModel.IterationSetupRowViewModels.FirstOrDefault(x => x.FrozenOnDate == "Active");
```

Comment:
Don't compare the `FrozenOnDate` but rather some property in the `Thing` of that row.